GOVERNANCE.md

```diff
@@ -33,12 +35,13 @@ A PR should be created with all relevant changes to update the repository for th
 It should apply the following actions:
 
 1. Change the "Unreleased version" heading in the CHANGELOG.md file
-   to the appropriate version name (e.g. "Version 1.2.3")
-   and add a new "Unreleased version" section heading above it;
+   to the appropriate version name and date (e.g. "Version 1.2.3 - 2020-12-31")
```

Comment:
I believe it would make sense to replicate the changes from e2c026c42639c998d399513126bf918d7aaea153 in this PR, so that they're introduced in lockstep with these guideline changes. WDYT @Xymph?

Reply:
I'm not really biased either way, but if you are splitting off commits from #102 anyway, I'm fine with that commit going into this PR too. Along with the initial header change per my comment above.

Reply:
I can't cleanly cherry-pick that commit into this branch, because it includes adding the section header for the new release (which should remain in #102), but I'll recreate the parts that need to be moved and will mark you as co-author of the commit :)

Reply:
Fine, if I need to make changes to my branch locally, and pull/push stuff around, please detail any steps.

Reply:
Yes, no worries. In fact, I should be able to update your branch myself. Then, if it looks good to you on GitHub, you will be able to pull the updated branch to your local clone as well.